GPU/GPUTracking/Global/GPUChainTrackingClusterizer.cxx

```diff
@@ -627,6 +627,7 @@ int32_t GPUChainTracking::RunTPCClusterizer(bool synchronizeOutput)
     uint32_t maxClusters = 0;
     int32_t deviceId = -1;
     int32_t numLanes = GetProcessingSettings().nTPCClustererLanes;
+    int32_t maxThreads = mRec->MemoryScalers()->nTPCdigits / 6000;
```

Comment:
@davidrohr Is this the right number of threads to use as the maximum?

Reply:
@ChSonnabend : on CPU or on GPU? On GPU you can do what you want. On CPU you should run `GPUReconstructionProcessing::getNKernelHostThreads(true)` threads.

Reply:
I mean, on the GPU itself use as many threads as you want, but if running on the GPU, please use only a single CU thread, as you wrote above.

Reply:
Perfect, thats the funciton I was looking for, thanks